workitem_test.go

```diff
@@ -44,9 +44,9 @@ func TestGetWorkItem(t *testing.T) {
 	repo := models.NewWorkItemRepository(ts)
 	controller := WorkitemController{ts: ts, wiRepository: repo}
 	payload := app.CreateWorkitemPayload{
-		Name: "foobar",
 		Type: "1",
 		Fields: map[string]interface{}{
+			"system.title":"Test WI",
 			"system.owner": "aslak",
```

Comment:
The `owner` field has been renamed to `assignee` in #185. Remember to change it once this gets merged.